### ISSUE TITLE
Finalize Recurring User Comms Test

### ISSUE THREAD

**Issue by chris-kuryak** [CDCgov/prime-reportstream#15189]
## Problem statement
ReportStream needs to learn the best way to maintain contact with users to continue providing them value. A recurring monthly update is one idea to explore to meet that need.

## What you need to know

 [Links to documents, workflows, or list of key people and contact info]

- [content draft](https://cdc.sharepoint.com/:w:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7BA20011D3-FA27-4A78-AC3B-334290195BBB%7D&file=Monthly%20update%20drafting.docx&action=default&mobileredirect=true&wdsle=0)

## Acceptance criteria

- [x] Finalize content/design in SendGrid
- [x] Approval from Max
- [x] Align with product on initial receiver list(s) and time 

## Decisions
- Initial Receiver List:
  - 
- Send date to initial list:
- Send date to remaining list:

**Comment by audreykwr** [comment]
@chris-kuryak FYI because of Tim schedule and timeline we might need to bump PO approval to a separate ticket

**Comment by chris-kuryak** [comment]
@audreykwr Ok -- should we make the separate ticket now, or is there still time to see if he'll get back to us in time?

**Comment by audreykwr** [comment]
Probably just make that ticket now, and if he does end up giving feedback in time, hooray.

**Comment by chris-kuryak** [comment]
Separating out the Approval by Tim (#15292) since he may not be around by the end of this sprint to approve.

**Comment by audreykwr** [comment]
Internal test and round 1 lists in [this sheet] under those tab labels (https://cdc.sharepoint.com/:x:/r/teams/ReportStream/Shared%20Documents/Design%20and%20Research/03_Content/Monthly%20update%20list.xlsx?d=w4a59252488bb4a11b23bde417777a86e&csf=1&web=1&e=95G94X)

**Comment by audreykwr** [comment]
Blocked by CDC domain issues, to get the test sent

**Comment by chris-kuryak** [comment]
Time seems TBD based on SendGrid issues.

**Comment by chris-kuryak** [comment]
Removing 

- Perform internal test with internal stakeholders and integrate feedback
- Initial receiver list email scheduled

To following ticket since we are blocked by SendGrid issues
